pkg/build/raw_test.go

```diff
@@ -14,17 +15,9 @@ import (
 
 func TestCreateRawImageCopyCommand(t *testing.T) {
 	// Setup
-	builder := Builder{
-		context: &image.Context{
-			ImageConfigDir: "config-dir",
-			ImageDefinition: &image.Definition{
-				Image: image.Image{
-					BaseImage:       "base-image",
-					OutputImageName: "build-image",
-				},
-			},
-		},
-	}
+	ctx, teardown := setupContext(t)
+	defer teardown()
+	builder := Builder{context: ctx}
```

Comment:
Same comment as one of the above. If we don't use the directories, why do we need to build a context? The main reason behind not using `setupContext()` is that it will eventually be easier to get rid of once we figure out whether the structure of the `build` package is what we want it to be.

Reply:
We do use the directories; `builder.generateBaseImageFilename()` and `builder.generateOutputImageFilename()` both use them.

Reply:
Those use string concatenation - I see nothing wrong with hardcoding these values in the tests.